release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2809,6 +2809,40 @@ To save the `vmcore` file, use the `crashkernel` setting to reserve 1024 MB of m
 
 * NFS volumes exported from VMware vSAN Files cannot be mounted by clusters running {product-title} 4.19 due to RHEL-83435. To avoid this issue, ensure that you are running VMware ESXi and vSAN at the latest patch versions of 8.0 P05, or later. (link:https://issues.redhat.com/browse/OCPBUGS-55978[OCPBUGS-55978])
 
+Known Gateway API issues
+
+* By default, `istiod` will create a ConfigMap `istio-ca-root-cert` in every namespace it watches, which can create unnecessary ConfigMaps. (link:https://issues.redhat.com/browse/OSSM-9076[OSSM-9076])
+
+* When using Gateway API via the Cluster Ingress Operator (CIO), any Istio specific resources (e.g. VirtualService) are not allowed or supported. (link:https://issues.redhat.com/browse/OSSM-9154[OSSM-9154])
+
+* Istio will copy all labels and annotations from your Gateway objects to the child resources it creates. (link:https://issues.redhat.com/browse/OSSM-8989[OSSM-8989])
```

Comment:
What problem does this create? Why is a release note needed for this? Should we have a KCS for this instead? 

The same applies for the next ~7 line items.

Reply:
@sferich888 we are basically trying to tell people - we know these are issues so please don't report duplicate issues.  We're trying to get ahead of bug reports, in other words.  If this is not the approach, please let me know.

Reply:
Do the majority of issues in SM come from customer direct filed bug reports? If not; I think it might be better to speak with CEE about why so many bug reports are being filed (that the team is having to close as duplicates). 

What is the duplicate bug count (and/or closure rate); compared to the overall backlog of issues the team is managing? cc: @CFields651 (as you might be needed to help look into this).

Reply:
@sferich888 I was more concerned with the unnecessary effort that the support person would need to go through to finally get to the point where they open a bug, and then we find that it is for an issue we already know about.  Or a customer, who, for example, makes grand plans to enable Gateway API on BareMetal without realizing it won't work.

I think the most important items can be summarized and added to existing 4.19 docs instead of release notes.  No doubt they will be more visible there anyway.  

@jmanthei I spoke to my team leads/manager, and I will try to come up with something like a "Things you need to know before enabling Gateway API" section.  For now, you can close this PR.   My apologies for the false start and thanks for the effort.